module/zfs/vdev_raidz.c

```diff
@@ -1739,10 +1633,13 @@ raidz_parity_verify(zio_t *zio, raidz_map_t *rm)
 	return (ret);
 }
 
+/* Make a kstat out of this? */
+#if 0
 /*
  * Keep statistics on all the ways that we used parity to correct data.
  */
 static uint64_t raidz_corrected[1 << VDEV_RAIDZ_MAXPARITY];
```

Comment:
@behlendorf I found this statistics that's otherwise unused. Not really sure how useful it is.
Any suggestion what to do with it?